solana/program_v2/packages/tests/src/network-suite/create-network.test.ts

```diff
@@ -37,6 +37,13 @@ describe('Gateway v2 Client', () => {
       },
       programProvider
     );
+    //   export declare type GatewayServiceOptions = {
```

Comment:
Think we can remove these commented out lines?

Reply:
I'd think so!